app/models/template.rb

```diff
@@ -1,7 +1,8 @@
 class Template < ActiveRecord::Base
   validates_lengths_from_database
 
-  attr_accessible :name, :template, :snippet, :audit_comment, :location_ids, :organization_ids, :locked, :default
```

Comment:
> The attr can still be set at creation, but not mass-assigned, that's all. The fix you put in should still work.

Yes, but not one-step creation, `JobTemplate.create(template)` which is what I meant, it's cleaner.

> As it's in the API docs then it'd need to be added back to permit mass-assignment.

Ok, I'll open a PR